### ISSUE TITLE
investigate an option to remove container itself on device disconnect 

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/mcloud-agent#116]
current approach remove containers forcibly for any disconnected devices and re-create from scratch during connect.
approach is great but in some cases we can't handle valid device id serial on disconnect due to the udev related issues.

in this case device is disconnected, containers exited/running and next connect doesn't work as expected because containers already exists.

to avoid complicated debugging with udev on all kind of Linux distributive let' investigate such shema:
1. create new containers during connect
2. `exit 0` from containers on disconnect (/dev/device... symlink is an option)
3. organize automatic restart only after failure
4. during connect start existing containers or create new ones if missed

**Comment by vdelendik** [comment]
Unable to keep device containers in Exited state because after physical disconnect 
1. adb connection is lost without possibility to reanimate
2. linked --device is lost  without possibility to reanimate

As result we can concentrate on making container removal independent of udev rule and ID_SERIAL detection. Updated ticket subject accordingly

**Comment by vdelendik** [comment]
too many issues with it. moved to 2.3 where we can do one more attempt for that

**Comment by vdelendik** [comment]
closing as won't fix. we already fixed it by adding container re-creation on start. so old container might exists in exited state and it is recreated during next connect when new usb connect symlink is generated
